generators/chipyard/src/main/scala/config/fragments/ClockingFragments.scala

```diff
@@ -122,3 +122,7 @@ class WithNoTileResetSetters extends Config((site, here, up) => {
 class WithNoResetSynchronizers extends Config((site, here, up) => {
   case ChipyardPRCIControlKey => up(ChipyardPRCIControlKey).copy(enableResetSynchronizers = false)
 })
+
+class WithNoClockTap extends Config((site, here, up) => {
```

Comment:
Need a one line comment. Fragment name is not clear enough

Reply:
I diagree for this specific case, but for the sake of consistency, I acquiesce